apps/slack/README.md

```diff
@@ -101,10 +104,14 @@ This sections explains how to run the Slack app locally.
   - Enable config location
   - Eneable request verification and store the secret in `lambda/config/serverless.dev.yml` (`signingSecret` -> `signing_secret`)
   - Create a key pair. Save the private key at `lamba/private-key.pem`
-- Configure Slack
+- Configure Slack (https://api.slack.com/apps)
   - Store Client ID and Client Secret (Basic Information -> App Credentials) in `lambda/config/serverless.dev.yml` (`oauthCredentials` -> `client_id` / `client_secret`)
   - Store Signing Secret (Basic Information -> App Credentials) in `lambda/config/serverless.dev.yml` (`slackSigningSecret` -> `signing_secret`)
 - Add the ngrok URL to in the `lambda/config/serverless.dev.yml` (`customDomain` -> `domainName`). Remove the `https://` from the URL. Add a trailing `/dev` to the URL (e.g.: `domainName: be25-95-91-246-99.ngrok.io/dev`)
+- `cd` into lambda
+- run `npm install && npm run start` in /lambda
+
+// Ignore below
```

Comment:
Yeah the docker file is wrong as are these instructions.

FYI if you intended this as an invisible comment you do you need to do `[//]`?